homeassistant/components/tado/climate.py

```diff
@@ -500,49 +514,49 @@ def _async_update_zone_data(self):
         self._current_tado_swing_mode = self._tado_zone_data.current_swing_mode
 
     @callback
-    def _async_update_zone_callback(self):
+    def _async_update_zone_callback(self) -> None:
         """Load tado data and update state."""
         self._async_update_zone_data()
         self.async_write_ha_state()
 
     @callback
-    def _async_update_home_data(self):
+    def _async_update_home_data(self) -> None:
         """Load tado geofencing data into zone."""
         self._tado_geofence_data = self._tado.data["geofence"]
 
     @callback
-    def _async_update_home_callback(self):
+    def _async_update_home_callback(self) -> None:
         """Load tado data and update state."""
         self._async_update_home_data()
         self.async_write_ha_state()
 
-    def _normalize_target_temp_for_hvac_mode(self):
+    def _normalize_target_temp_for_hvac_mode(self) -> None:
+        def adjust_temp(min_temp, max_temp):
```

Comment:
Why is this an inline function?
It should be a util function IMHO

Reply:
The only use case is within this function. That's why I put it inline.

Reply:
Pleas ad typing to it